### ISSUE TITLE
Duplicated step search results

### ISSUE THREAD

**Issue by koral--** [bitrise-io/bitrise-workflow-editor#127]
As can be seen on attached screenshot. Link to the bitrise app: 
https://www.bitrise.io/app/1d190f1c0e4c8486#/workflow

![zrzut ekranu 2017-05-17 o 12 19 46](https://cloud.githubusercontent.com/assets/3340954/26149635/46fbac56-3afb-11e7-9aff-a4b5d9ce9fd7.png)

**Comment by erosdome** [comment]
Thanks for reporting, will look into it ASAP.

**Comment by erosdome** [comment]
Could not reproduce it, although found a possible cause, added a fix for it. Could you check if it is resolved?

https://discuss.bitrise.io/t/offline-workflow-editor-workflow-editor-v2-v1-0-4-released/1765

**Comment by koral--** [comment]
I can confirm that it is resolved now.

**Comment by viktorbenei** [comment]
Awesome, thanks for confirming @koral-- ! ;)

**Comment by koral--** [comment]
I saw this again, now I remember exact scenario.

Steps to reproduce:
1. Go to workflows tab.
2. Go to bitrise.yml tab.
3. Repeat few times.

Step list is concatenated with itself each time:
![image](https://user-images.githubusercontent.com/3340954/27614386-983f34fe-5ba1-11e7-9463-47275fb36ff3.png)

**Comment by viktorbenei** [comment]
@koral-- can you please upgrade to the latest workflow editor version (`bitrise plugin update workflow-editor`) and try it again? I couldn't reproduce it with the latest version.

**Comment by koral--** [comment]
I've used online editor on bitrise.io and could not repro it either now.
Has something relevant changed since my previous comment or maybe we are still missing some element?

**Comment by viktorbenei** [comment]
I'd say maybe a recently deployed fix solved this, or of course it might be that there's one more step required to reproduce the issue..

I'll close the issue, but if you (or us) could reproduce this again we'll reopen it. In any case, thanks for reporting @koral-- ! :)

**Comment by koral--** [comment]
@viktorbenei I encountered that again.
Here is the affected app: https://www.bitrise.io/app/1d190f1c0e4c8486#
Steps which I performed:

1. open app page from dashboard
2. add new empty workflow
3. add few new steps to it

There was a loader/waiting indicator in step list pop-up displayed for about 10 seconds when I've tried to add new steps.
So maybe this the key information: clicking on + button when step list is still loading?
